shard-operator/pom.xml

```diff
@@ -195,12 +193,17 @@
         <activeByDefault>true</activeByDefault>
       </activation>
       <properties>
-        <namespace>default</namespace>
-        <quarkus.openshift.namespace>${namespace}</quarkus.openshift.namespace>
         <quarkus.openshift.deploy>false</quarkus.openshift.deploy>
         <quarkus.profile>openshift</quarkus.profile>
         <event-bridge.k8s.platform>openshift</event-bridge.k8s.platform>
       </properties>
+      <dependencies>
```

Comment:
@vaibhavjainwiz Is this change related?

Reply:
we need to add below dependency to generate openshift specific file.
 ```
 <dependency>
          <groupId>io.quarkus</groupId>
          <artifactId>quarkus-openshift</artifactId>
          <optional>true</optional>
        </dependency>
```

Reply:
https://quarkus.io/guides/deploying-to-openshift#bootstrapping-the-project

Reply:
@vaibhavjainwiz Sorry I think I'm missing too much context to fully understand this PR. 

Is there some process related to our CPaaS builds that needs the additional of this dependency to generate something specific to Openshift?

thanks,
Rob